basics/02-Vis-Basics/02-Vis-Basics.Rmd

```diff
@@ -8,6 +8,17 @@ runtime: shiny_prerendered
 tutorial:
   id: "com.rstudio.primers.02-vis-basics"
   version: 1.0
+params:
```

Comment:
Is the idea that we want these to show up in the "knit with parameters" menu in the IDE and on connect? Specifically, would we lose anything if we hard-coded a glossary YAML field into leanr docs?

Reply:
:+1: to using `glossary` as a key instead of `params` - I'll make the fix.

Reply:
Yes, `params` is the only field that currently makes values available inside of a document. I don't think we should overload it, because the glossary use doesn't fit well with the `params` workflow.

I'm suggesting that we file an issue with learnr to build out a new field. I haven't seen as far as you here, so I'm not sure that these terms need to be called from within the tutorial to be defined---but I don't foresee anything that would prevent that, once the feature has been added to learnr (rmarkdown?).

Alternatively, we may be able to add a standard but optional glossary section to every tutorial (similar to the Table of Contents), or use the glossary terms as we use the course description (i.e. on a splash page that directs users into the tutorial).

Does this sound worth filing an issue to you? If so I'm happy to take the lead or add my support to the thread, whichever you prefer.